cernopendata/templates/cernopendata_records_ui/docs/detail.html

```diff
@@ -12,10 +12,10 @@ <h1> {{ record.title }} </h1>
                     <small>{{ record.author}} </small>
                     </p>
                     <p class="badges-box">
-                    {% if record.type %} <a class="badge badge-type" href="/search?type={{ record.type.primary }}">{{ record.type.primary }}</a>
+                    {% if record.type %} <a class="ui label badge-type" href="/search?type={{ record.type.primary }}">{{ record.type.primary }}</a>
                     {% if record.type.secondary %}
                     {% for type in record.type.secondary %}
-                    <a class="badge badge-subtype" href="/search?type={{ record.type.primary }}&subtype={{ type }}">{{ type }}</a>
+                    <a class="ui label badge-subtype" href="/search?type={{ record.type.primary }}&subtype={{ type }}">{{ type }}</a>
```

Comment:
Since we have custom styles for `.badge-subtype` the hover effect looks a bit strange due to the black text:

![image](https://user-images.githubusercontent.com/8863238/106005860-eeb2b100-60b4-11eb-9dd4-e34857871ff3.png)

I see two options:

1. Add `color: white` on `:hover` to all the custom badge classes we have.

https://github.com/cernopendata/opendata.cern.ch/blob/d81b9439c33df4a5b7b7cd16428bc75012d6da21/cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss#L403

2. Use [semantic-ui classes](https://semantic-ui.com/elements/label.html#colored) that handle this so we have less code to maintain, and eventually remove all these custom classes. In this particular case `ui label blue` would have a similar appearance, as `badge-subtype` is using a blue color.

Let me know what you think

https://github.com/cernopendata/opendata.cern.ch/blob/d81b9439c33df4a5b7b7cd16428bc75012d6da21/cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss#L414

Reply:
I fully agree about solution (2.), perhaps we can tackle this separately since changing this would probably mean changing the colors at global level.